### PR TITLE
Use css font loading if It can.

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/rpgtkoolmv/corescript/issues/19

### CODE REVIEW

File: js/rpg_core/Graphics.js [hunk 1]
```diff
@@ -801,6 +829,9 @@ Graphics._createRenderer = function() {
     } catch (e) {
         this._renderer = null;
     }
+
```
Comment (krmbn0576) [inline js/rpg_core/Graphics.js]: Is this related to the font?

Comment (ghost) [reply js/rpg_core/Graphics.js]: mixed.. I'll remove it.

**Comment by krmbn0576** [comment]
Reviewed. Nice! :heart:
